calico_containers/pycalico/datastore.py

```diff
@@ -76,8 +76,10 @@
 BGP_HOST_PEERS_PATH = BGP_HOST_PATH + "peer_v%(version)s/"
 BGP_HOST_PEER_PATH = BGP_HOST_PATH + "peer_v%(version)s/%(peer_ip)s"
 
-# Grabs hostname from etcd datastore key
-HOSTNAME_DATASTORE_RE = re.compile(BGP_HOSTS_PATH + "(.*)/ip_addr_v[46]")
+# Grabs hostname from etcd datastore keys
+HOSTNAME_IP_ETCD_RE = re.compile(BGP_HOSTS_PATH + "(.*)/ip_addr_v[46]")
```

Comment:
The word datastore is preferred to etcd. What was your reasoning behind changing it?